static/panes/compiler.js

```diff
@@ -1184,6 +1184,8 @@ Compiler.prototype.setAssembly = function (result, filteredCount) {
     this.assembly = asm;
     if (!this.outputEditor || !this.outputEditor.getModel()) return;
     var editorModel = this.outputEditor.getModel();
+    if (result.languageId !== 'undefined')
+        editorModel._languageId = result.languageId;
```

Comment:
I don't know if it's OK to just reach into `monaco`'s private fields and change things. I'd like to find out if there's an official way to change language. I think it's something like:
```suggestion
        this.outputEditor.setModelLanguage(editorModel, lang);
```

but it may not be that simple (see Microsoft/monaco-editor#375)